server/src/controller/master.controller.ts

```diff
@@ -58,55 +52,92 @@ export const getAvailableMasters = async (req: Request, res: Response) => {
         
     try {
         const { currentOrderId, city_id, start_work_on, clock_id } = req.query
-        
-
-        const installDuration = await db.query('SELECT installation_time FROM clocks WHERE id = $1', [clock_id])
-
-        if(installDuration.rows.length) {
-
-            const { installation_time } = installDuration.rows[0]
-            let date = new Date(`${start_work_on}`)
-            date.setUTCHours(date.getHours() + installation_time)
-            const end_work_on = date.toISOString()
-
-            let readBookedMasters
-
-            if(currentOrderId){
-
-                readBookedMasters = await db.query(`SELECT master_id FROM orders 
-                                                            WHERE ((start_work_on <= $1 AND end_work_on >= $1) 
-                                                            OR (start_work_on <= $2 AND end_work_on >= $2)) 
-                                                            AND id != $3`, [start_work_on, end_work_on, currentOrderId])
-            
-            } else {
 
-                readBookedMasters = await db.query(`SELECT master_id FROM orders 
-                                                            WHERE ((start_work_on <= $1 AND end_work_on >= $1) 
-                                                            OR (start_work_on <= $2 AND end_work_on >= $2))`, [start_work_on, end_work_on])
-            
+                const installationDuration = await Clock.findOne({
+                    attributes: ['installation_time'],
+                    where: {
+                        id: clock_id
+                    }
+                })
+
+                if(installationDuration) {
+
+                    const { installation_time } = installationDuration
```

Comment:
some values are camel case, some of them are snake case. We're prefering the camel case when using js